digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -439,7 +439,22 @@ CommandStatus createNextCommandStatus(
             // finish this poll once and wait finish marker in head of this method in next poll, considering risk of crushing in this poll.
             nextStatus.put("task_finished_at", Instant.now().getEpochSecond());
             // Set exit code of container finished to nextStatus
-            nextStatus.put("status_code", task.getContainers().get(0).getExitCode());
+            int containerExitCode = task.getContainers().get(0).getExitCode().intValue();
```

Comment:
Just in case, can we expect `getExitCode()` always return non-null value ?

Reply:
I assume so because at least current code assumes non-null value in 
https://github.com/treasure-data/digdag/blob/v0_11/digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java#L347